Error: EACCES: permission denied, open '/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/index.js'

I was exploring Github's actions and wanted to `Firebase-Distribution-Github-Action` after using the sample action I am getting the following error. It did work for 2 times & suddenly stopped working without making any changes. Did I missed anything to add OR configure?


Workflow File
```
# This is a basic workflow to help you get started with Actions
name: Firebase QA App Distribution

# Controls when the action will run. Triggers the workflow on push or pull request
# events but only for the master branch
on:
  push:
    branches: [ master ]

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
      - uses: actions/checkout@v1

      # setup JDK.1.8
      - name: set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8

      # Prepare QA build
      - name: build release
        run: sudo ./gradlew assembleDebug

      # Distribute APK to firebase App Distribution
      - name: upload artifact to Firebase App Distribution
        uses: wzieba/Firebase-Distribution-Github-Action@v1.2.1
        with:
          appId: ${{secrets.FIREBASE_QA_APP_ID}}
          token: ${{secrets.FIREBASE_TOKEN}}
          groups: internal-test
          file: app/build/outputs/apk/debug/app-debug.apk
```


```
2020-06-21T16:47:19.6829979Z Build container for action use: '/home/runner/work/_actions/wzieba/Firebase-Distribution-Github-Action/v1.2.1/Dockerfile'.

2020-06-21T16:48:19.3104692Z ##[command]/usr/bin/docker build -t c27d31:613707ed5f1740928f76d78bdec856e2 "/home/runner/work/_actions/wzieba/Firebase-Distribution-Github-Action/v1.2.1"
2020-06-21T16:48:19.3386929Z Sending build context to Docker daemon   12.8kB
2020-06-21T16:48:19.3387474Z 
2020-06-21T16:48:19.3710051Z Step 1/6 : FROM node:12.10.0-alpine
2020-06-21T16:48:19.3712024Z  ---> ef7d474eab14
2020-06-21T16:48:19.3713730Z Step 2/6 : WORKDIR /app
2020-06-21T16:48:19.3715215Z  ---> Using cache
2020-06-21T16:48:19.3716587Z  ---> cee83b63b2ce
2020-06-21T16:48:19.3718112Z Step 3/6 : COPY . /app
2020-06-21T16:48:19.3719850Z  ---> Using cache
2020-06-21T16:48:19.3721820Z  ---> 9fe2c3d69d62
2020-06-21T16:48:19.3734617Z Step 4/6 : RUN npm install -g firebase-tools     && apk update     && apk add git
2020-06-21T16:48:19.3958354Z  ---> Running in 333d0f1ae47d
2020-06-21T16:48:21.3283264Z [91mnpm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
2020-06-21T16:48:38.3340192Z [0m/usr/local/bin/firebase -> /usr/local/lib/node_modules/firebase-tools/lib/bin/firebase.js
2020-06-21T16:48:38.4942415Z 
2020-06-21T16:48:38.4942825Z > protobufjs@6.9.0 postinstall /usr/local/lib/node_modules/firebase-tools/node_modules/protobufjs
2020-06-21T16:48:38.4943001Z > node scripts/postinstall
2020-06-21T16:48:38.4943079Z 
2020-06-21T16:48:38.5540364Z 
2020-06-21T16:48:38.5540780Z > storage-engine@3.0.7 postinstall /usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine
2020-06-21T16:48:38.5540963Z > node ./compatibility/detect.js
2020-06-21T16:48:38.5541070Z 
2020-06-21T16:48:38.5994297Z [91mfs.js:130
2020-06-21T16:48:38.5994513Z     throw err;
2020-06-21T16:48:38.5994684Z     ^
2020-06-21T16:48:38.5994765Z 
2020-06-21T16:48:38.5994964Z Error: EACCES: permission denied, open '/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/index.js'
2020-06-21T16:48:38.5995194Z     at Object.openSync (fs.js:451:3)
2020-06-21T16:48:38.5995373Z     at Object.writeFileSync (fs.js:1276:35)
2020-06-21T16:48:38.5995570Z     at /usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/detect.js:35:15
2020-06-21T16:48:38.5995798Z     at Object.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/detect.js:45:3)
2020-06-21T16:48:38.5996008Z     at Module._compile (internal/modules/cjs/loader.js:936:30)
2020-06-21T16:48:38.5996201Z     at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
2020-06-21T16:48:38.5996397Z     at Module.load (internal/modules/cjs/loader.js:790:32)
2020-06-21T16:48:38.5996588Z     at Function.Module._load (internal/modules/cjs/loader.js:703:12)
2020-06-21T16:48:38.5996784Z     at Module.require (internal/modules/cjs/loader.js:830:19)
2020-06-21T16:48:38.5996966Z     at require (internal/modules/cjs/helpers.js:68:18) {
2020-06-21T16:48:38.5997134Z   errno: -13,
2020-06-21T16:48:38.5997291Z   syscall: 'open',
2020-06-21T16:48:38.5997447Z   code: 'EACCES',
2020-06-21T16:48:38.5997635Z   path: '/usr/local/lib/node_modules/firebase-tools/node_modules/storage-engine/lib/compatibility/index.js'
2020-06-21T16:48:38.5997822Z }
2020-06-21T16:48:39.2997879Z [0m[91mnpm WARN [0m[91moptional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/firebase-tools/node_modules/chokidar/node_modules/fsevents):
2020-06-21T16:48:39.2998161Z [0m[91mnpm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
2020-06-21T16:48:39.3006563Z [0m[91mnpm WARN ws@7.3.0 requires a peer of bufferutil@^4.0.1 but none is installed. You must install peer dependencies yourself.
2020-06-21T16:48:39.3007086Z npm WARN ws@7.3.0 requires a peer of utf-8-validate@^5.0.2 but none is installed. You must install peer dependencies yourself.
2020-06-21T16:48:39.3007264Z [0m[91m
2020-06-21T16:48:39.3232107Z [0m[91mnpm ERR! code[0m[91m ELIFECYCLE
2020-06-21T16:48:39.3232300Z [0m[91mnpm ERR! errno 1
2020-06-21T16:48:39.3238777Z [0m[91mnpm[0m[91m ERR! storage-engine@3.0.7 postinstall: `node ./compatibility/detect.js`
2020-06-21T16:48:39.3239318Z npm[0m[91m ERR! Exit status 1
2020-06-21T16:48:39.3239477Z npm[0m[91m ERR! 
2020-06-21T16:48:39.3239632Z npm ERR! Failed at the storage-engine@3.0.7 postinstall script.
2020-06-21T16:48:39.3239815Z npm[0m[91m ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-06-21T16:48:39.3499524Z [0m[91m
2020-06-21T16:48:39.3499723Z npm [0m[91mERR! A complete log of this run can be found in:
2020-06-21T16:48:39.3500159Z npm ERR!     /root/.npm/_logs/2020-06-21T16_48_39_342Z-debug.log
2020-06-21T16:48:40.1330726Z The command '/bin/sh -c npm install -g firebase-tools     && apk update     && apk add git' returned a non-zero code: 1
2020-06-21T16:48:40.1331885Z [0m
2020-06-21T16:48:40.1451003Z ##[error]Docker build failed with exit code 1
2020-06-21T16:48:40.1492582Z Cleaning up orphan processes
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,9 @@
-FROM node:12.10.0-alpine
+FROM node:14.4.0-alpine3.12
 
 WORKDIR /app
 COPY . /app
 
-RUN npm install -g firebase-tools \
+RUN yarn global add firebase-tools \
     && apk update \
     && apk add git 
 

--- a/README.md
+++ b/README.md
@@ -59,7 +59,7 @@ jobs:
     - name: build release 
       run: ./gradlew assembleRelease
     - name: upload artifact to Firebase App Distribution
-      uses: wzieba/Firebase-Distribution-Github-Action@v1.2.1
+      uses: wzieba/Firebase-Distribution-Github-Action@v1
       with:
         appId: ${{secrets.FIREBASE_APP_ID}}
         token: ${{secrets.FIREBASE_TOKEN}}

